homeassistant/components/shelly/strings.json

```diff
@@ -138,21 +124,27 @@
       }
     },
     "sensor": {
+      "gas_detected": {
+        "state": {
+          "none": "None",
+          "mild": "Mild",
+          "heavy": "Heavy",
+          "test": "Test"
+        }
+      },
       "operation": {
         "state": {
           "warmup": "Warm-up",
           "normal": "Normal",
           "fault": "Fault"
-        },
-        "state_attributes": {
-          "self_test": {
-            "state": {
-              "not_completed": "Not completed",
-              "completed": "Completed",
-              "running": "Running",
-              "pending": "Pending"
-            }
-          }
+        }
+      },
+      "self_test": {
+        "state": {
+          "not_completed": "Not completed",
+          "completed": "Completed",
+          "running": "Running",
+          "pending": "Pending"
```

Comment:
We should add translations for `options` attribute.

Reply:
Good point